### DISCUSSION TITLE
第3节课作业(1班)

### DISCUSSION THREAD

**Discussion by AXYZdong** [InternLM/Tutorial#196] [作业]
**基础作业：**

复现课程知识库助手搭建过程 (截图)

**进阶作业：**

选择一个垂直领域，收集该领域的专业资料构建专业知识库，并搭建专业问答助手，并在 [OpenXLab](https://openxlab.org.cn/apps) 上成功部署（截图，并提供应用地址）

**Comment by xujinzh** [comment]
https://xujinzh.github.io/2024/01/08/ai-Interlm-langchain-RAG/

**Comment by woxinyonghen** [comment]
https://github.com/woxinyonghen/InternLMCamp/tree/main/homework02/基础作业/

**Comment by biscuit279** [comment]
https://blog.csdn.net/qq_46212981/article/details/135488056

**Comment by Gu-Xiao** [comment]
https://zhuanlan.zhihu.com/p/676955063  基础作业完成，进阶作业资源正在申请中

**Comment by ustbzgn** [comment]
https://blog.csdn.net/qq_46052444/article/details/135491939

**Comment by Xpg74138** [comment]
https://github.com/Xpg74138/InternLMCamp/blob/main/Second_homework.ipynb

**Comment by RongjianLiang** [comment]
https://zhuanlan.zhihu.com/p/677022214
(作业截图在文章的最后一节）

**Comment by blurmemo** [comment]
https://github.com/blurmemo/LLM/blob/main/note/InternLM/homework/chap2-homework.md

**Comment by zpc-666** [comment]
https://blog.csdn.net/zpc1219/article/details/135489868
作业截图在文章的6.5小节

**Comment by strong-leaf** [comment]
大模型(书生.浦语大模型)finetune实战_lesson3笔记与作业 - warm green的文章 - 知乎
https://zhuanlan.zhihu.com/p/677011273
(备注：作业在最后部分)

**Comment by UncleLLD** [comment]
https://blog.csdn.net/uncle_ll/article/details/135492002

**Comment by Andy1314Chen** [comment]
https://www.bilibili.com/read/cv29275301/?jump_opus=1

**Comment by 10086zgyd** [comment]
https://github.com/10086zgyd/InternLMLangChain

**Comment by gpt-dance** [comment]
作业在最后
https://zhuanlan.zhihu.com/p/676732671

**Comment by raulming** [comment]
https://quilled-apparatus-d9a.notion.site/505cdf23b70f4bcf8a7eea3b78298bd0 见末尾作业截图

**Comment by AXYZdong** [comment]
https://nuly9zxzf1.feishu.cn/docx/B4rFdjWMuoiQXwxqqVuc5waynlb?ccm_open_type=lark_docs_home_Recent&from=tab_home#part-S9wHdRXvnoLd1WxO9yHcTAybnid

**Comment by gongxiucheng** [comment]
https://ckfajwu6zv.feishu.cn/docx/GuhFdjcnCoMJaZxCePbcA3zdnLg

**Comment by grey2818** [comment]
https://zhuanlan.zhihu.com/p/677553043

**Comment by lds-coder** [comment]
https://zhuanlan.zhihu.com/p/677646719

**Comment by xiaozhuang223** [comment]
https://zhuanlan.zhihu.com/p/677140526
